code/modules/mob/observer/freelook/marker/marker.dm

```diff
@@ -0,0 +1,148 @@
+
+/*
+	The marker is the heart of the necromorph army
+*/
+/obj/machinery/marker
```

Comment:
Probably best to add a smaller marker as a subtype of this using the smaller sprites.